### ISSUE TITLE
NaNs in output of `DistributedEmbedding.preprocess()`

### ISSUE THREAD

**Issue by abheesht17** [keras-team/keras-rs#141]
Reproduced this here: https://colab.research.google.com/gist/abheesht17/57575eaf845c248cbf1e760e1332a97c/distributed_embedding_jax.ipynb.

Is this expected and just for padding?

**Comment by abheesht17** [comment]
@cantonios, @silkyarora

**Comment by abheesht17** [comment]
Hmmm, probably this: https://github.com/jax-ml/jax-tpu-embedding/blob/main/jax_tpu_embedding/sparsecore/lib/core/input_preprocessing_util.cc#L83-L94. Never mind
